### PR TITLE
Sara edits

### PR DESCRIPTION
**Pull request by sastoudt**
- a few small edits to formatting, etc.
- added map for smilodon
- added some activities
- proposed an alternative to creating `extinct_tol_list`

### CODE REVIEW

File: r/neotoma_tutorial.Rmd [hunk 1]
```diff
@@ -382,6 +474,7 @@ extinct_mammals$taxon.name <- trimws(extinct_mammals$taxon.name)
 ## and also requires manual interaction
 ## you can download data set below
 ## Manual selection key: 1, 0, 2, 0, 2, 1 
+## [TODO] I needed one more number for the manual selection. I put NA and then get 26 not classified below.
```
Comment (sastoudt) [inline r/neotoma_tutorial.Rmd:475]: Can you double check this?

File: r/neotoma_tutorial.Rmd [hunk 2]
```diff
@@ -396,32 +489,24 @@ extinct_tol <- extinct_tol[!is.na(extinct_tol)]
 
 ### Make data frame to merge with 
 
-[TODO] THis is an old way of making a dataframe.  Is there a more efficient way? This is super unreadable and confusing code. Maybe skip this part and upload the dataframe after?
+[TODO] Switch from read/write to using this if you think this approach is ok.
```
Comment (sastoudt) [inline r/neotoma_tutorial.Rmd:490]: If you think this version is okay, we can make eval = T and avoid reading in an intermediate file.

File: r/neotoma_tutorial.Rmd [hunk 3]
```diff
@@ -396,32 +489,24 @@ extinct_tol <- extinct_tol[!is.na(extinct_tol)]
...
-  names(y) <- c("taxon.name", "order", "family", "genus")
-  extinct_tol_list[[i]] <- y
-
+  y <- c(y, as.character(xNice$name))
+  return(y)
 }
 
-
+## see why I'm removing 74 and 200 below
+## [TODO] Check that these numbers stay the same after we deal with the discrepancy in manual selection key
```
Comment (sastoudt) [inline r/neotoma_tutorial.Rmd:503]: I'll have to double check this once we figure out the manual selection discrepancy.

File: r/neotoma_tutorial.Rmd [hunk 4]
```diff
@@ -551,12 +650,18 @@ The majority of the taxa is made up of Mastodon samples [*Mammut americanum*](ht
...
 ## Incorporation of More Data
 
-I wanted to see if I could make sense of the habitat of these species or climate during the Pliocene to map onto the plot, but unfortunatley I could not find mappable data. I wish I had [this papers](https://www.researchgate.net/publication/37456896_Modelling_Middle_Pliocene_warm_climates_of_the_USA) data. This is a question I had many times during these this analysis.  Does anyone know where we can get predicted ancient global temperature? 
+I wanted to see if I could make sense of the habitat of these species or climate during the Pliocene to map onto the plot, but unfortunatley I could not find mappable data. I wish I had [this paper's](https://www.researchgate.net/publication/37456896_Modelling_Middle_Pliocene_warm_climates_of_the_USA) data. This is a question I had many times during these this analysis.  Does anyone know where we can get predicted ancient global temperature? 
 
 As we explore more databases we hope to gain a better understanding of the data available and how they can be combined to provide insights into our natural world. 
 
 Stay tuned for more exploration!
 
+## Activity
```
Comment (sastoudt) [inline r/neotoma_tutorial.Rmd:657]: Do you have a contribution guide/workflow for the website?

File: r/neotoma_tutorial.Rmd [hunk 5]
```diff
@@ -22,16 +22,17 @@ Note: If you are an SQL wizard, you are in luck, this database is extensive and
 If you are working on your own machine, you must first install these packages.
 
 ## Set Up
-```{r results='hide', warning=FALSE}
+```{r results='hide', warning=FALSE,message = F}
 # If you are working on your own machine, you must first install these packages.
-# install.packages(c("ggplot", "ggmap", "tidyverse", "skimr", "taxize", "neotoma"))
+# install.packages(c("ggplot", "ggmap", "tidyverse", "skimr", "taxize", "neotoma", "WikipediR"))
```
Comment (sastoudt) [inline r/neotoma_tutorial.Rmd:27]: For Binder: I added a package.

File: r/neotoma_tutorial.Rmd (hunk 4 above)
Comment (iamciera) [reply r/neotoma_tutorial.Rmd:657]: I don't! I was thinking of updating the README.md for the website.  I put it on my to do list!

File: r/neotoma_tutorial.Rmd (hunk 1 above)
Comment (iamciera) [reply r/neotoma_tutorial.Rmd:475]: I'm not sure we should include this whole part in the tutorial. It is a little crazy.  It's not our fault, just the API and taxize together is pretty clunky....I do want to include somehow though.  I might just have two notebooks and we will go through the shorter one with the data read in option for the workshop.  Then I will have the longer one if they are interested.

Comment (sastoudt) [reply r/neotoma_tutorial.Rmd:475]: I like that idea. It'll save us time, but then they'll have access to the full pipeline.

Comment (iamciera) [reply r/neotoma_tutorial.Rmd:475]: It' a plan!

**Pull request merged**